subsys/bluetooth/host/testing.h

```diff
@@ -0,0 +1,27 @@
+/**
+ * @file testing.h
+ * @brief Internal API for Bluetooth testing.
+ */
+
+/*
+ * Copyright (c) 2017 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#if defined(CONFIG_BT_TESTING)
+struct bt_test_cbs {
```

Comment:
lets be consistent with rest of BT API so name it struct bt_test_cb,   some for other names cbs->cb

Reply:
done